doc/ref_model/chapters/chapter04.md

```diff
@@ -262,6 +263,32 @@ Table 4-10 shows performance measurement capabilities.
 
 <p align="center"><b>Table 4-11:</b> Cloud Infrastructure management Resource Management Measurements</p>
 
+<a name="4.1.7"></a>
+### 4.1.7 Acceleration/Offload API Requirements 
+HW Accelerators and Offload functions with abstracted interfaces are preferred and can functionally be interchanged, but their characteristics might vary and it is likely that the CNFs/VNFs and Virtualization Infrastructure will have certification requirements for the different implementations. It is also often needed to have a SW implementation of these functions with the same abstracted interfaces for the deployment situations when there are no more HW Accelerator or Offload functions available.
+ 
+For Accelerators and Offload functions with externally exposed differences in their capabilities or management functionality these differences must be clear through the management API either explicit for the differing functions or implicit through the use of a unique APIs.
+ 
+Regardless of exposed or internal different capabilities and characteristics, the operators generally require a choice of multiple implementations also for Accelerators and Offload function realization, which drive the need for ease of portability in between implementations and vendors.
```

Comment:
operators are telco operators here? Can you please clarify?

Reply:
Yes, it's the Telco Operators. I will improve the text to clarify that.